typings/eslint/index.d.ts

```diff
@@ -18,7 +18,8 @@ export namespace Scope {
     scopes: Scope[]
     globalScope: Scope | null
     acquire(node: VAST.ESNode | VAST.Program, inner?: boolean): Scope | null
-    getDeclaredVariables(node: VAST.ESNode): Variable[]
+    /** Added in ESLint v8.38.0 */
```

Comment:
```suggestion
    /** @since ESLint v8.38.0 */
```

(also for the other occurrences)